server/opts.go

```diff
@@ -398,6 +398,15 @@ func (o *Options) ProcessConfigFile(configFile string) error {
 	errors := make([]error, 0)
 	warnings := make([]error, 0)
 
+	// First check whether a system account has been defined, as that
+	// is a condition for other features to be enabled.
+	var hasSystemAccount bool
```

Comment:
We now do a quick check on whether the system account has been enabled, since rest of config map items will be processed randomly